int main()
{
int x;
try {x++;} 
catch ("test",x) {}
finally { return 0;}
return 1;
}
